backend/core/src/listings/listings-csv-exporter.service.ts

```diff
@@ -0,0 +1,163 @@
+        City: listing.property.buildingAddress?.city,
+        State: listing.property.buildingAddress?.state,
+        Zip: listing.property.buildingAddress?.zipCode,
+        Year_Built: listing.property.yearBuilt,
+        Neighborhood: listing.property.neighborhood,
+        Region: listing.property.region,
+        Latitude: listing.property.buildingAddress?.latitude,
+        Longitude: listing.property.buildingAddress?.longitude,
+        Home_Type: listing.homeType,
+        Accept_Section_8: listing.section8Acceptance,
+        Number_Of_Unit_Groups: listing.unitGroups?.length,
+        Community_Types: listing.listingPrograms
+          ?.map((listingProgram) => listingProgram.program.title)
+          .join(", "),
+        Application_Fee: listing.applicationFee,
+        Deposit_Min: listing.depositMin,
+        Deposit_Max: listing.depositMax,
+        Deposit_Helper: listing.depositHelperText,
+        Costs_Not_Included: listing.costsNotIncluded,
+        Utilities_Included: Object.entries(listing.utilities ?? {})
+          .filter((entry) => entry[1] === true)
+          .map((entry) => entry[0])
+          .join(", "),
+        Property_Amenities: listing.property.amenities,
+        Additional_Accessibility_Details: listing.property.accessibility,
+        Unit_Amenities: listing.property.unitAmenities,
+        Smoking_Policy: listing.property.smokingPolicy,
+        Pets_Policy: listing.property.petPolicy,
+        Services_Offered: listing.property.servicesOffered,
+        Accessibility_Features: Object.entries(listing.features ?? {})
+          ?.filter((entry) => entry[1] === true)
+          .map((entry) => entry[0])
+          .join(", "),
+        Grocery_Stores: listing.neighborhoodAmenities?.groceryStores,
+        Public_Transportation: listing.neighborhoodAmenities?.publicTransportation,
+        Schools: listing.neighborhoodAmenities?.schools,
+        Parks_and_Community_Centers: listing.neighborhoodAmenities?.parksAndCommunityCenters,
+        Pharmacies: listing.neighborhoodAmenities?.pharmacies,
+        Health_Care_Resources: listing.neighborhoodAmenities?.healthCareResources,
+        Credit_History: listing.creditHistory,
+        Rental_History: listing.rentalHistory,
+        Criminal_Background: listing.criminalBackground,
+        Building_Selection_Critera: listing.buildingSelectionCriteria,
+        Required_Documents: listing.requiredDocuments,
+        Important_Program_Rules: listing.programRules,
+        Special_Notes: listing.specialNotes,
+        Review_Order: listing.reviewOrderType,
+        Lottery_Date: dayjs(listing.events?.startTime).format("MM-DD-YYYY"),
+        Lottery_Start: dayjs(listing.events?.startTime).format("MM-DD-YYYY h:mm:ssA"),
+        Lottery_End: dayjs(listing.events?.endTime).format("MM-DD-YYYY h:mm:ssA"),
+        Lottery_Notes: listing.events[0]?.note,
+        Application_Due_Date: listing.applicationDueDate,
+        Waitlist: listing.isWaitlistOpen,
```

Comment:
Confirming with product if this is the correct field or if its intended to be "displayWaitlist"